pontoon/base/tests/test_placeables.py

```diff
@@ -0,0 +1,16 @@
+from django_nose.tools import assert_equal
+
+from pontoon.base.tests import TestCase
+from pontoon.base.utils import NewlineEscapePlaceable
+
+
+class PlaceablesTests(TestCase):
+    def test_newline_escape_placeable(self):
+        """Test detecting newline escape sequences"""
+        placeable = NewlineEscapePlaceable
+
+        assert_equal(placeable.parse(u'A string\\n')[1], placeable([u'\\n']))
```

Comment:
This is basically testing if your regex is correct, which I'm not terribly concerned about testing. To me, the thing to test here is `mark_placeables`, specifically if it is given input with newlines does it output the correct HTML tags.